web_tree_float_head/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Display float head in tree view
+=====================================
+
+This module set head as float in tree view.
```

Comment:
Ok, work is in progess.

Best: This module sets the header always visible in the tree view .